ansible_collections/arista/avd/roles/eos_designs/python_modules/overlay/router_path_selection.py

```diff
@@ -178,13 +180,17 @@ def _get_local_interfaces_for_path_group(self, path_group_name: str) -> list | N
 
         return local_interfaces
 
-    def _get_dynamic_peers(self) -> dict | None:
+    def _get_dynamic_peers(self, is_local_pg, ipsec) -> dict | None:
```

Comment:
let's have only one boolean entry for this which is `disable_ipsec`
```suggestion
    def _get_dynamic_peers(self, disable_ipsec:bool) -> dict | None:
```